random1.py

```diff
@@ -0,0 +1,6 @@
+import random
+import logging
```

Comment:
**Suggestion:** It's a good practice to configure the logging level and format. This will help to control the level of details you want to log and also make the logs more readable.
```suggestion
import logging
logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```